bindgen/ir/comp.rs

```diff
@@ -663,12 +682,22 @@ where
         unfilled_bits_in_unit = data_size - unit_size_in_bits;
     }
 
+    let final_bit_size = if let Some(next_field) = next_non_bitfield {
+        next_field
+            .offset()
+            .map_or(unit_size_in_bits, |off| off - bitfields_bit_begin_off)
+    } else if let Some(parent_layout_) = parent_layout {
+        parent_layout_.size * 8 - bitfields_bit_begin_off
```

Comment:
Same for this 8, I'm assuming the 8 comes from 64 bits? But that's not correct on non-64-bit architectures is it? Should it use the word size?

Reply:
Same as before